diff --git a/node_modules/@dylankenneally/react-native-ssh-sftp/android/build.gradle b/node_modules/@dylankenneally/react-native-ssh-sftp/android/build.gradle
--- a/node_modules/@dylankenneally/react-native-ssh-sftp/android/build.gradle
+++ b/node_modules/@dylankenneally/react-native-ssh-sftp/android/build.gradle
@@ -2,7 +2,7 @@
 buildscript {
     repositories {
-        jcenter()
         google()
+        mavenCentral()
     }
 
     dependencies {
@@ -29,7 +29,6 @@
 
 repositories {
     mavenCentral()
-    jcenter()
 }
 
 dependencies {
@@ -35,8 +34,10 @@ dependencies {
-    implementation 'org.bouncycastle:bcprov-jdk15on:1.70'
-
     api 'com.facebook.react:react-native:+'
     api 'com.github.mwiede:jsch:0.2.17'
     api 'com.fasterxml.jackson.core:jackson-core:2.10.3'
     api 'com.fasterxml.jackson.core:jackson-databind:2.10.3'
 }
+
+configurations.all {
+    exclude group: 'org.bouncycastle', module: 'bcprov-jdk15on'
+}

